0.6/docker-entrypoint.sh

```diff
@@ -40,8 +40,13 @@ fi
 # CONSUL_CONFIG_DIR isn't exposed as a volume but you can compose additional
 # config files in there if you use this image as a base, or use CONSUL_LOCAL_CONFIG
 # below.
-CONSUL_DATA_DIR=/consul/data
-CONSUL_CONFIG_DIR=/consul/config
+if [ -z "$CONSUL_DATA_DIR" ]; then
```

Comment:
You can drop these lines as they are always going to be set due to the ENV vars set in the dockerfile. Unless the user does something silly like: -e CONSUL_DATA_DIR="" ... and should we cater for said user? I think they should learn from their evil ways..